pkg/networking/networking.go

```diff
@@ -52,11 +53,34 @@ type defaultHeadersRoundTripper struct {
 	networkAccess            *networkImpl
 }
 
+func (rt *defaultHeadersRoundTripper) logRoundTrip(request *http.Request, response *http.Response) {
+	logHeader := http.Header{}
+	loglevel := zerolog.TraceLevel
+
+	if rt.networkAccess.logger.GetLevel() == loglevel {
+		for i, v := range request.Header {
+			for _, value := range v {
+				if strings.ToLower(i) == "authorization" || strings.ToLower(i) == "session-token" {
+					value = strings.Split(v[0], " ")[0] + " ***"
+				}
+				logHeader.Add(i, value)
+			}
+		}
+	}
+
+	rt.networkAccess.logger.WithLevel(loglevel).Msgf("> request: %s, %s, %v", request.Method, request.URL.String(), logHeader)
```

Comment:
Question: Why not: `... logger.Trace()`? Why make the log-level configurable? I think it may not be needed here :)

Reply:
this is a performance improvement. loglevel is also used to decide wether or not to prepare a special version of the http header for logging.